WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -8,6 +8,7 @@ open class QuickStartTourGuide: NSObject {
     private var suggestionWorkItem: DispatchWorkItem?
     private weak var recentlyTouredBlog: Blog?
     static let notificationElementKey = "QuickStartElementKey"
+    private let identifier = UUID().uuidString
```

Comment:
Quick Start is currently assumed to be a singleton because of `QuickStartTourGuide.find()`. But if it ends up not being so in the future, we might have some trouble dismissing the correct notices because of the random `UUID` value. I'd recommend just changing this to `"QuickStartTourGuide"`. 

Let's also make use of `Notice.Tag` for clarity. 

```suggestion
private let noticeTag: Notice.Tag = "QuickStartTourGuide"
```

What do you think? 🙂

Reply:
Looks great.